pkg/controller/account/account_controller.go

```diff
@@ -269,18 +263,16 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		if currentAcctInstance.Spec.ManualSTSMode {
 			accountClaim, acctClaimErr := r.getAccountClaim(currentAcctInstance)
 			if acctClaimErr != nil {
-				reqLogger.Error(acctClaimErr, "unable to get accountclaim for sts account")
+				reqLogger.Error(acctClaimErr, "Unable to get AccountClaim for STS Account")
 				utils.SetAccountClaimStatus(
```

Comment:
Same comment here as well, with the unhandled Error for the same function, that is `utils.SetAccountClaimStatus()`. You can fix it like this:

```go
				err := utils.SetAccountClaimStatus(
					r.Client,
					reqLogger,
					accountClaim,
					"Failed to get AccountClaim for CSS account",
					"FailedRetrievingAccountClaim",
					awsv1alpha1.ClientError,
					awsv1alpha1.ClaimStatusError,
				)
				if err != nil {
					return reconcile.Result{}, err
				}
```